components/client/src/main/java/com/hotels/styx/client/netty/connectionpool/NettyConnectionFactory.java

```diff
@@ -57,19 +56,15 @@
     private Bootstrap bootstrap;
 
     private NettyConnectionFactory(Builder builder) {
-        this.eventLoopFactory = builder.eventLoopFactory();
+        this.eventLoopFactory = eventLoopFactory(builder.name, builder.clientWorkerThreadsCount);
         this.httpConfig = requireNonNull(builder.httpConfig);
         this.sslContext = builder.tlsSettings == null ? null : SslContextFactory.get(builder.tlsSettings);
 
-        this.httpRequestOperationFactory = builder.httpRequestOperationFactory != null
-                ? builder.httpRequestOperationFactory
-                : (HttpRequest request) -> new HttpRequestOperation(
-                            request,
-                            builder.originStatsFactory,
-                            builder.flowControlEnabled,
-                            builder.responseTimeoutMillis,
-                            builder.requestLoggingEnabled,
-                            builder.longFormat);
+        this.httpRequestOperationFactory = requireNonNull(builder.httpRequestOperationFactory);
+    }
+
+    private static ClientEventLoopFactory eventLoopFactory(String name, int threadCount) {
```

Comment:
This method seems a little redundant. If the class name is too long, perhaps we can change it? Or create a static-factory method.